index.js

```diff
@@ -4,14 +4,22 @@ const _ = require('lodash'),
   path = require('path'),
   files = require('nymag-fs');
 
-function getSiteConfig(site) {
-  const ymlPath = path.resolve(site.dir, 'anf.yml').replace('.yml', ''); // files.getYaml adds the file extension for some reason, so remove it here
+var log = require('./services/log').setup({ file: __filename });
 
-  return files.getYaml(ymlPath);
+function getSiteConfig(site) {
+  const ymlPath = path.resolve(site.dir, 'anf.yml');
+
+  if (files.fileExists(ymlPath)) {
+    return files.getYaml(ymlPath.replace('.yml', '')); // files.getYaml adds the file extension for some reason, so remove it here
+  } else {
+    log('error', { message: 'No anf.yml config file found for this site' });
```

Comment:
For logging, the second argument to the `log` function is the message, so you don't need to declare an object and pass in the `message` property.

The third argument is an object where you can append any data you want to send across.